src/MenuBar.tsx

```diff
@@ -1,10 +1,11 @@
-import { Collapse } from "@mui/material";
+import { Collapse, type CollapseProps } from "@mui/material";
 import { makeStyles } from "tss-react/mui";
+import type { Except } from "type-fest";
 import { Z_INDEXES, getUtilityClasses } from "./styles";
 
 export type MenuBarClasses = ReturnType<typeof useStyles>["classes"];
 
-export type MenuBarProps = {
+export type MenuBarProps = Except<CollapseProps, "children"> & {
```

Comment:
This has the slightly confusing consequence of including props which are always ignored/overridden, namely `in` and `unMountOnExit`. Those should be explicitly handled, either by excluding them from this type definition (like you did for `children`), or better yet allowing users to override default behavior if they provide them. For instance, setting `unmountOnExit=true` when destructuring props, thereby allowing users to override it